plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -156,11 +156,16 @@ int kmtIoctl(int fd, unsigned long request, void *arg)
 
 static void free_e(CriuKfd *e)
 {
-	for (int i = 0; i < e->n_bo_info_test; i++) {
-		if (e->bo_info_test[i]->bo_rawdata.data)
-			xfree(e->bo_info_test[i]->bo_rawdata.data);
-		if (e->bo_info_test[i])
-			xfree(e->bo_info_test[i]);
+	for (int i = 0; i < e->n_bo_entries; i++) {
+		if (e->bo_entries[i]) {
```

Comment:
This seems to be fixing a previous bug. Nothing wrong with that, since it's going to get squashed anyway. Just making sure I'm not missing anything.

Reply:
Yes